Updates-2.0-UI/u2-sce-test-wso2am-3.2.0-full-testgrid.yaml

```diff
@@ -0,0 +1,39 @@
+# TestGrid job configuration
+
+version: '0.9'
+emailToList: "integration-builder@wso2.com,rosen@wso2.com,prasanna@wso2.com,
+              nandika@wso2.com"
+infrastructureConfig:
+  iacProvider: CLOUDFORMATION
+  infrastructureProvider: AWS
+  containerOrchestrationEngine: None
+  includes:
+    - CentOS-7.5
+    - MySQL-5.7
+    - ADOPT_OPEN_JDK11
+  provisioners:
+    - name: two-node-deployment
+      remoteRepository: "git@github.com:wso2-support/aws-apim.git"
+      remoteBranch: "3.2.0-u2"
```

Comment:
Shouldn't this be `3.2.0-u2-staging`? Or are we creating a separate job for that?